### PR TITLE
[3632][IMP] report_common_nrq 

### PR DESCRIPTION
**Pull request by kanda999**
[3672](https://www.quartile.co/web#id=3672&menu_id=505&cids=3&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: report_common_nrq/report/common_template.xml [hunk 1]
```diff
@@ -175,6 +175,16 @@
                                         </t>
                                     </td>
                                 </tr>
+                                <tr t-if="doc.company_id.vat">
+                                    <td style="width:100;">VAT:</td>
```
Comment (yostashiro) [inline report_common_nrq/report/common_template.xml]: I think VAT may not be a good word choice.

```suggestion
                                    <td style="width:100;">Registration No.:</td>
```

**Review by yostashiro — COMMENTED** [review]
How about tax adjustments on report_sale_nrq and report_purchase_nrq?

**Comment by yostashiro** [comment]
> 請求書だけでよい

Never mind.  I see this in the meeting notes.

**Comment by yostashiro** [comment]
> 旧のほうはもう使用しておらず、必要ないため削除

How about this, though?

**Comment by kanda999** [comment]
> > 旧のほうはもう使用しておらず、必要ないため削除
> 
> How about this, though?

I have created another task and will follow up wih it.

File: report_common_nrq/report/common_template.xml [hunk 2]
```diff
@@ -175,6 +175,16 @@
...
                                     </td>
                                 </tr>
+                                <tr t-if="doc.company_id.vat">
+                                    <td style="width:100;">Registration No.</td>
+                                    <td>
+                                        <t>
+                                            <span
+                                                t-field="doc.company_id.vat"
+                                            />
+                                        </t>
```
Comment (yostashiro) [inline report_common_nrq/report/common_template.xml]:
```suggestion
                                        <span t-field="doc.company_id.vat" />
```

**Review by yostashiro — APPROVED** [review]

**Pull request merged**